spec/integration/http.spec.js

```diff
@@ -90,13 +105,28 @@ describe( 'HTTP', function() {
 
 		harness.addResource( {
 			name: 'test',
+			middleware: [
+				function( envelope, next ) {
+					if( envelope.headers.accept ) {
+						var accept = envelope.headers.accept;
+						var match = /[.]v([0-9]*)/.exec( accept );
```

Comment:
@digitalBush It's in a test for differentiating between handle calls. How I got the version was less important but I can change it.